code/modules/urist/items/clothstuff.dm

```diff
@@ -94,3 +94,46 @@
 				else
 					to_chat(user, "<span class='warning'>\The [src] is used up, but there are more wounds to treat on \the [affecting.name].</span>")
 			use(used)
+
+/obj/item/loom
+	name = "table loom"
+	desc = "A loom small enough to only take up a table instead of the whole floor."
+	icon = 'icons/urist/items/wood.dmi'
+	icon_state = "loom"
+
+/datum/reagent/cottonfiber
+	name = "cotton fiber"
+	description = "A mass of cotton fibers."
+	taste_description = "cotton"
+	reagent_state = SOLID
+	color = "#ffffff"
```

Comment:
It's usually not a good idea to use 100% saturation colors, doesn't leave much colorspace to work with for blending.
```suggestion
	color = "#f0f0f0"
```